_pages/install/integrations/cds-integration.md

```diff
@@ -0,0 +1,239 @@
+{{ site.version_note_part1 }} 1.5 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+## Overview
+
+The SAP Commerce Cloud, Context-Driven Services solution provides real-time customer experience personalization for SAP Commerce. You can integrate Context-Driven Services with your Spartacus Storefront, including the Profile Tag and the Merchandising Carousel features.
+
+For more information, see [SAP Commerce Cloud, Context-Driven Services](https://help.sap.com/viewer/product/CONTEXT-DRIVEN_SERVICES/SHIP/en-US) on the SAP Help Portal.
+
+## Requirements
+
+To integrate Context-Driven Services with Spartacus, release **1905.9** of Commerce Cloud is required.
+
+Also, the Anonymous Consent feature in Spartacus needs to be enabled. For more information, see [Anonymous Consent]({{ site.baseurl }}{% link _pages/dev/features/anonymous-consent.md %}).
+
+## Enabling Context-Driven Services in Spartacus
+
+To enable Context-Driven Services in Spartacus, you need to configure both the Commerce Cloud back end, and the Spartacus front end.
+
+### Configuring the Back End for Context-Driven Services
+
+The following steps describe how to add custom headers to your CORS settings, as well as how to define a consent template that allows events to be sent.
+
+1. Add the `x-profile-tag-debug` and `x-consent-reference` custom headers to `corsfilter.ycommercewebservices.allowedHeaders`. 
+
+    If you are using the Assisted Service Module, add these custom headers to `corsfilter.assistedservicewebservices.allowedHeaders` as well.
+
+    For more information, see [Configuring CORS](https://sap.github.io/cloud-commerce-spartacus-storefront-docs/installing-sap-commerce-cloud/#configuring-cors).
+
+2. Define a consent template with an ID of `PROFILE`, which will allow events to be sent.
+
+    You can define the consent template by importing the following ImpEx:
+
+    ```sql
+    $lang=en
+    INSERT_UPDATE ConsentTemplate;id[unique=true];name[lang=$lang];description[lang=$lang];version[unique=true];baseSite(uid)[unique=true,    default=electronics-spa];exposed
+    ;PROFILE;"Allow SAP Commerce Cloud, Context-Driven Services tracking";"We would like to store your browsing behavior so that our website can dynamically present you with a personalized browsing experience and our customer support agents can provide you with contextual customer support.";1;;true
+    ```
+
+### Configuring Spartacus for Context-Driven Services
+
+You can carry out all of the following steps after you have set up your Spartacus Storefront. For more information, see [Building the Spartacus Storefront from Libraries]({{ site.baseurl }}{% link _pages/install/building-the-spartacus-storefront-from-libraries.md %}).
+
+1. Install the Context-Driven Services library by running the following command from within the root directory of your storefront app: 
+
+    ```bash
+    npm i @spartacus/cds@1.0.0-next.0
```

Comment:
Just a note here: once the npm package has a stable version, we should remove this '@1.0.0-next.0`. I will open a PR once this happens

Reply:
Good point! As soon as it's stable, let me know and I will udpate this 🙂